base/src/main/java/com/smartdevicelink/proxy/interfaces/ISdl.java

```diff
@@ -208,6 +208,7 @@
      * @param systemCapabilityType a system capability type that should be retrieved
      * @return the system capability provided if available, null if not
      */
+    @Deprecated
```

Comment:
Please add javadoc as to what method should be used instead of this one because of deprecation.

Reply:
I will add that to deprecated methods here and in SCM